lighthouse-core/report/report-generator.js

```diff
@@ -56,7 +56,7 @@ class ReportGenerator {
 
     // Converts a name to a link.
     Handlebars.registerHelper('nameToLink', name => {
-      return name.toLowerCase().replace(/\s/, '-');
+      return name.toLowerCase().replace(/\s/g, '-');
```

Comment:
Yea, that's probably best for now. We really don't have a great way to test the registered handlebar helpers atm. The few tests we have, look at the generated output :\ 

Maybe make the asset more specific so there is a smaller chance of false positives in the future. We want make sure "progressive-web-app" is from the `nameToLink` output and not something else. There's also [another place to check](https://github.com/GoogleChrome/lighthouse/blob/5fd1cfeaee82b6484e824a61d0a9f36bd06074c7/lighthouse-core/report/templates/report-template.html#L78):

    assert.ok(/class="menu__link" href="#progressive-web-app"/gm.test(html));
    assert.ok(/id="progressive-web-app"/gm.test(html));